### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by sap-prajapati**

### CODE REVIEW

**Comment by anilrao316** [comment]
please add screenshot ,videos as part of pull request description also add one liner mentioning about the application u r building

File: Week-2/1st Assignment/index.html [hunk 1]
```diff
@@ -44,38 +44,50 @@ <h1 class="block_third_heading">
             </p>
             <br>
             <a class="block_third_link" href="http://www.lipsum.com/">Link</a>
-        </div>
+        </article>
 
     </section>
 
         <form class="Form">
```
Comment (anilrao316) [inline Week-2/1st Assignment/index.html:51]: small f in form

File: Week-2/1st Assignment/index.html [hunk 2]
```diff
@@ -7,12 +7,12 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div class="TopHeader">
+    <header class="TopHeader">
```
Comment (anilrao316) [inline Week-2/1st Assignment/index.html:10]: follow BEM naming conventions

**Comment by sap-prajapati** [comment]
169364077-16b12e65-f64b-4665-b6c5-b105b283cad0
169592365-8a34defa-bd40-44bb-8e92-1cdb4a64a380
169592436-5f9559c5-e8d0-4fce-8017-0a6cf9981a54

https://user-images.githubusercontent.com/40159886/169869108-fde74954-10be-4ce9-a764-070aff1b2f77.mp4

**Pull request merged**